homeassistant/components/sky_remote/config_flow.py

```diff
@@ -0,0 +1,39 @@
+"""Config flow for sky_remote."""
+
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.selector import selector
+
+from .const import CONF_LEGACY_CONTROL_PORT, DOMAIN
+
+DATA_SCHEMA = {
+    vol.Required(CONF_HOST): cv.string,
+    vol.Required(CONF_NAME): cv.string,
```

Comment:
Names as config are not allowed. Please use the host as the default name (the user can rename everything in the UI anyway)

```suggestion
```